scripts/bulk_index.pl

```diff
@@ -49,32 +49,40 @@ BEGIN
 
 $indexer->create_index;
 
+my $run_time = time;
 my $loop = IO::Async::Loop->new;
 my $timer = IO::Async::Timer::Periodic->new(
     first_interval => 0,
     interval       => 15,
     reschedule     => 'skip',
 
     on_tick => sub {
+        printf "Running after %d seconds\n", time - $run_time;
         my $start_users = time;
         say "indexing users" if $verbose;
-        $indexer->bulk_load('Bugzilla::User');
-        print "    ", time - $start_users, " seconds\n" if $verbose > 1;
+        my $users = $indexer->bulk_load('Bugzilla::User');
+        bulk_load_stats($start_users, $users) if $verbose > 1;
```

Comment:
I'm slightly surprised to see you calling bulk_load_stats() directly here. Shouldn't that be covered by $indexer->bulk_load() for you? Or, is the goal here to only collect these stats at *this* call to bulk_load, but not *all* calls?